### PR TITLE
Fix failing integration tests

### PR DESCRIPTION
**Pull request by marcellp**
# Background
#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [ ] Tested locally with Inframock (with latest production data downloaded with biomage experiment pull)
- [ ] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

File: .github/workflows/run-e2e.yaml [hunk 1]
```diff
@@ -79,34 +63,35 @@ jobs:
...
-        name: Copy experiments
+      - id: set-up-creds
+        name: Set up Kubernetes credentials.
         run: |-
-          cd biomage-utils/
-          for EXP_ID in $(yq e '.experiment-ids' ../config.yaml -j | jq '.[]')
-          do
-            biomage experiment copy -e $(echo $EXP_ID | sed s'/"//g')
-          done
+          aws eks update-kubeconfig --name biomage-$ENV
```
Comment (marcellp) [inline .github/workflows/run-e2e.yaml:76]: We need to make sure this is saved in `iac` somewhere. Currently the rights of the CI runner are manually assigned by me.

File: .github/workflows/run-e2e.yaml [hunk 2]
```diff
@@ -119,6 +104,17 @@ jobs:
           K8S_ENV: ${{ github.event.inputs.environment || 'production' }} 
           SANDBOX_ID: ${{ github.event.inputs.sandboxId }}
 
+      - id: enable-cronjob
```
Comment (marcellp) [inline .github/workflows/run-e2e.yaml:107]: This needs to run always (add `if: always()`).

**Comment by marcellp** [comment]
Check why error occurs here:

https://github.com/biomage-ltd/testing/runs/4382637905?check_suite_focus=true

**Review by aerlaut — APPROVED** [review]
LGTM. A new ticket has to be created to move credentials to IAC.

**Pull request merged**